bigbluebutton-html5/imports/ui/components/actions-bar/service.js

```diff
@@ -8,13 +9,24 @@ const getBreakouts = () => Breakouts.find({ parentMeetingId: Auth.meetingID })
   .fetch()
   .sort((a, b) => a.sequence - b.sequence);
 
-const getUsersNotAssigned = (users) => {
-  const breakouts = getBreakouts();
-  const breakoutUsers = breakouts
-    .reduce((acc, value) => [...acc, ...value.users], [])
-    .map(u => u.userId);
-  return users.filter(u => !breakoutUsers.includes(u.intId));
-};
+const getBreakoutUser = user => Users.find({
+  extId: new RegExp(user.userId),
```

Comment:
What is the reason for the `RegExp` here?

Reply:
Because the Breakout user has an `extId` with a extra number E.G:

`Main user extId: jjj445o1`
`Breakout user extId: jjj445o1-1`

Reply:
Since the userId is always at the start of the breakout userId you would probably want a "^" at the start so the search is a little quicker.